### ISSUE TITLE
Javadoc @link tag highlighting broken when there are multiple on one line

### ISSUE THREAD

**Issue by tylertian123** [redhat-developer/vscode-java#958]
Javadoc `{@link}` tags are not highlighted properly when there are multiple of them on one line (only the first is highlighted and others are not).

##### Environment
- Operating System: Ubuntu 18.04
- JDK version: 11.0.3
- Visual Studio Code version: 1.36.0-insider
- Java extension version: 0.46.0

##### Steps To Reproduce
1. Open a Java file
2. Paste in the following Javadoc comment:
```java
/**
 * {@link String#charAt(int)} {@link String#chars()}
 * {@link String#toCharArray()} {@link String#codePointAt(int)}
 */
```
![image](https://user-images.githubusercontent.com/32781310/59814271-b8129300-92e2-11e9-85ec-a8e9661531a7.png)
Notice how only the first `{@link}` tag on each line is highlighted.

##### Current Result
Only the first `{@link}` tag is properly highlighted, the rest is just highlighted like a normal comment.

##### Expected Result
All `{@link}` tags should be highlighted.

**Comment by fbricon** [comment]
looks like an issue with upstream's textmate Java grammar

**Comment by Eskibear** [comment]
it has been fixed now in 1.40

<img width="506" alt="Screen Shot 2019-11-21 at 2 01 09 PM" src="https://user-images.githubusercontent.com/2351748/69311501-6d1cf300-0c67-11ea-90a6-b363055f8c30.png">

**Comment by fbricon** [comment]
Closing since this is fixed in latest VS Code
